tedana/config.py

```diff
@@ -0,0 +1,5 @@
+DEFAULT_ICA_METHOD = "robustica"
+DEFAULT_N_ROBUST_RUNS = 30
+DEFAULT_N_MAX_ITER = 500
+DEFAULT_N_MAX_RESTART = 10
+DEFAULT_SEED = 42
```

Comment:
Want to resolve absence of newlines at end of files before creating PR to TEDANA.

Reply:
Thanks. It is fixed by commit 4fc3043.